### PR TITLE
Rhdevdocs 6204: Creating a separate release note folder and a tile for docs.redhat.com for Builds 1.2

### PR DESCRIPTION
**Pull request by shivanisathe25**
Version(s):
`build-docs-1.2`

Issue:

Link to docs preview:

QE review:
- Doesn't require QE review

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -26,8 +26,6 @@ Name: About Builds
 Dir: about
 Distros: openshift-builds
 Topics:
```
Comment (eromanova97) [inline _topic_maps/_topic_map.yml:28]: So this is the current state:
```
---
Name: About Builds
Dir: about
Distros: openshift-builds
Topics:
- Name: Overview of Builds
  File: overview-openshift-builds
...
```
Add this for release notes:
```
---
Name: Release notes
Dir: release_notes
Distros: openshift-builds
Topics:
- Name: Release Notes
  File: ob-release-notes
Name: About Builds
...
```

File: release_notes/ob-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+For more information about {builds-shortname}, see xref:../about/overview-openshift-builds.adoc#overview-openshift-builds[Overview of {builds-shortname}].
+
+
+include::modules/ob-compatibility-support-matrix.adoc[leveloffset=+1]
+
+include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
+
+// Modules included, most to least recent
+
+include::modules/ob-release-notes-1-1.adoc[leveloffset=+1]
```
Comment (eromanova97) [inline release_notes/ob-release-notes.adoc:31]:
```suggestion
include::modules/ob-release-notes-1-2.adoc[leveloffset=+1]
```

Comment (eromanova97) [inline release_notes/ob-release-notes.adoc:31]: I am not sure based on the changes I see in this PR, so just to make sure:
* include the `ob-release-notes-1-2.adoc` file in the `modules` folder (module)
* include the  `ob-release-notes.adoc` file in the `release_notes` folder (assembly)

**Comment by eromanova97** [comment]
Hello @shivanisathe25 let me know when you apply the changes I suggested. I will then review once again :) Thank you!

File: modules/ob-release-notes-1-2.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:5]:
```suggestion
[id="ob-release-notes-1-2_{context}"]
```
This file needs changing everything from 1.1 to 1.2. I know you showed me those changes so maybe just the file did not get saved or something :smile:

File: modules/ob-release-notes-1-2.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-2_{context}"]
+= Release notes for {builds-shortname} General Availability 1.1
+
+{builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} 4.16.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:8]:
```suggestion
{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.16.
```

Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:8]:
```suggestion
{builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} <TBD>.
```
As I suppose the version for OCP might change

File: modules/ob-release-notes-1-2.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+[id="new-features-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:10]:
```suggestion
[id="new-features-1-2_{context}"]
```

File: modules/ob-release-notes-1-2.adoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.1.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:13]:
```suggestion
The following sections highlight what is new in {builds-shortname} 1.2.
```

File: modules/ob-release-notes-1-2.adoc [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+[id="builds-new-features-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:15]:
```suggestion
[id="builds-new-features-1-2_{context}"]
```

File: modules/ob-release-notes-1-2.adoc [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+* The builds controllers now use Tekton's V1 API to create and access the `TaskRun` that backs a `BuildRun`.
+
+* With this release, you can now define a build without any source. This is useful if you want to run the build using only the local source.
+
+* The output image section now supports an optional timestamp field to change the image creation timestamp. Use the `SourceTimestamp` string to set it to match the source timestamp.
+
+* With this release, the `.source.type` field is now required in both `build` and `buildRun`.
+
+* With this release, the Operator now installs Shipwright by default and introduces several user experience enhancements for Builds.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:26]: You cn delete the actual RNs and just keep this as a template for when the actual RNs will come :) 
```suggestion

```

File: modules/ob-release-notes-1-2.adoc [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+
+[id="csi-driver-new-features-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:29]:
```suggestion
[id="csi-driver-new-features-1-2_{context}"]
```

File: modules/ob-release-notes-1-2.adoc [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+=== Shared Resource CSI Driver
+
+* The Shared Resource CSI Driver is now generally available. With this release, the CSI Shared Driver enables sharing of `ConfigMaps` and secrets across different namespaces in Kubernetes clusters. The driver has permission to read all Kubernetes secrets. This feature improves resource efficiency by reducing duplication and simplifies configuration management in multi-tenant environments. Administrators can define access policies to control which namespaces can read or modify these shared resources.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:32]:
```suggestion
```
Same as previous comment

File: modules/ob-release-notes-1-2.adoc [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+[id="known-issues-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:34]:
```suggestion
[id="known-issues-1-2_{context}"]
```

File: modules/ob-release-notes-1-2.adoc [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+== Known issues
+
+The following section highlights known issues in {builds-shortname} 1.1.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:37]:
```suggestion
The following section highlights known issues in {builds-shortname} 1.2.
```

File: modules/ob-release-notes-1-2.adoc [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+[id="builds-known-issues-1-1_{context}"]
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:39]:
```suggestion
[id="builds-known-issues-1-2_{context}"]
```

File: modules/ob-release-notes-1-2.adoc [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+=== Builds
+
+* With this release, direct upgrades from Builds 1.0 to Builds 1.1 are not supported. To upgrade to Builds 1.1, you must delete the `ShipwrightBuild` object, uninstall the Builds `1.0.z` operator, and then install the Builds `1.1.0` operator from OperatorHub.
```
Comment (eromanova97) [inline modules/ob-release-notes-1-2.adoc:42]:
```suggestion
```

**Review by adellape — COMMENTED** [review]
And reminder to squash before merge-review.

**Pull request merged**